Tests/AK/TestAKMath.cpp

```diff
@@ -0,0 +1,2305 @@
+/*
+ * Copyright (c) 2023, the SerenityOS developers.
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <LibTest/TestCase.h>
+
+#include <AK/Array.h>
+#include <AK/Math.h>
+
+struct MathTest {
+    double input;
+    double expected_output;
+};
+
+// Generated with Python + numpy:
+// echo -e "import numpy as np\nfor x in np.arange(-1020, 1020, 0.9):\n    print(f'{{ {x}, {2**x} }},')" | python3
+// FIXME: Actually use these for a test of AK::pow; currently many inputs are broken due to our bad implementation.
```

Comment:
Let's not add something that's essentially a 2200+ line FIXME. If you insist on having a FIXME, just leave the intent and the code to generate the table please.